samcli/commands/build/command.py

```diff
@@ -128,6 +129,16 @@
     help="Path to environment variable json file (e.g., env_vars.json) to pass into build containers",
     cls=ContainerOptions,
 )
+@click.option(
+    "--container-dir-mount",
+    "-v",
+    default=None,
+    multiple=True,
+    required=False,
+    help="Directories to be mounted into docker container."
+    " E.g. sam build --use-container --container-dir-mount /host/path:/container/path",
```

Comment:
Since our builds support customizing different images for each build function, does it make more sense to also support function specific mount dir like --container-dir-mount Function1=/host/path:/container/path?